packages/p2p/src/components/composite-calendar/two-month-picker.jsx

```diff
@@ -0,0 +1,109 @@
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import React from 'react';
+import { Calendar } from '@deriv/components';
+import { addMonths, diffInMonths, epochToMoment, subMonths, toMoment } from '@deriv/shared';
+
+const TwoMonthPicker = React.memo(({ onChange, isPeriodDisabled, value }) => {
+    const [left_pane_date, setLeftPaneDate] = React.useState(subMonths(value, 1).unix());
+    const [right_pane_date, setRightPaneDate] = React.useState(value);
+
+    const navigateFrom = e => {
+        setLeftPaneDate(e.unix());
+        setRightPaneDate(addMonths(e, 1).unix());
+    };
+
+    /**
+     * Only allow previous months to be available to navigate. Disable other periods
+     *
+     * @param date
+     * @param range
+     * @returns {boolean}
+     */
+    const validateFromArrows = (date, range) => {
```

Comment:
I think we can also move these out to the utility file, keep all the logic there, also great documentation!